### PR TITLE
Add ability to pipe a command's output to dashcam cli

### PR DESCRIPTION
**Pull request by yassine-cc**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -90,7 +114,7 @@ program
     }
   });
 
-if (process.stdin.isTTY) {
+if (process.stdin.isTTY || process.argv[2] === "x") {
```
Comment (ianjennings) [inline index.js]: @yassine-cc does this mean this already worked?

File: README.md [hunk 2]
```diff
@@ -55,6 +55,14 @@ Anything you type in your terminal will appear in your dash. To exit, simply typ
 exit
 ```
 
+## Record command output
+
+To record the output of a command in the Dashcam app (In this example the `ping 1.1.1.1` command ), use the following command
+
+```
+ping 1.1.1.1 | dashcam x
```
Comment (ianjennings) [inline README.md]: Can we call this command `pipe` instead of `x`

**Review by ianjennings — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (yassine-cc) [reply index.js]: Yes

File: README.md (hunk 2 above)
Comment (yassine-cc) [reply README.md]: It's done

**Pull request merged**